### ISSUE TITLE
Nested array loading issue

### ISSUE THREAD

**Issue by GoogleCodeExporter** [runt18/google-bigquery#240]
```
What steps will reproduce the problem?
1. Please run load command:
bq load --source_format=NEWLINE_DELIMITED_JSON mytest.testload 
impsFailure.json.gz bq_table_schema.json

2. Got error:
Waiting on bqjob_r2ecf292a19ffb54c_0000014cdc9e5bd5_1 ... (49s) Current status: 
DONE
BigQuery error in load operation: Error processing job 
'my-project-id:bqjob_r2ecf292a19ffb54c_0000014cdc9e5bd5_1': Unexpected. Please 
try again.

What is the expected output? What do you see instead?
Data should be loaded but got error instead:
Console job history showing:
Error Reason:internalError. Get more information about this error at 
Troubleshooting Errors: internalError.
Errors:
Unexpected. Please try again.

What version of the product are you using? On what operating system?
Using:
Google Cloud SDK 0.9.56

bq 2.0.18
bq-nix 2.0.18
compute 2015.04.14
core 2015.04.15
core-nix 2014.10.20
dns 2015.04.14
gcloud 2015.04.14
gcloud-extensions-linux-x86_64 0.14.1
gcutil 1.16.5
gcutil-nix 1.16.5
gsutil 4.11
gsutil-nix 4.6
sql 2015.04.09

Please provide any additional information below.
Error is related to nested array: 
user_info.products_tags[].value[].offer_identifiers[]
after removing it bq load works fine.

```

Original issue reported on code.google.com by `arkadius...@rtbhouse.com` on 21 Apr 2015 at 3:45

Attachments:
- [bq_table_schema.json](https://storage.googleapis.com/google-code-attachments/google-bigquery/issue-240/comment-0/bq_table_schema.json)
- [impsFailure.json.gz](https://storage.googleapis.com/google-code-attachments/google-bigquery/issue-240/comment-0/impsFailure.json.gz)

**Comment by GoogleCodeExporter** [comment]
```
Hi, thanks for the error report, and please accept my apologies that you never 
got a response until now.  We're working through some revisions to our oncall + 
support policy.  If you ever have issues that languish like this, please feel 
free to reach out via Stack Overflow with a question tagged to 
"google-bigquery".

I don't know if this is still relevant for you or not, but I can repro the 
failure you're seeing, and have found the root cause; I'll have a fix shortly.
```

Original comment by `thomasp...@google.com` on 25 Nov 2015 at 10:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I've identified a fix for the underlying issue, but it's in a part of the 
system that we're in the midst of making other major improvements to.  How 
critical is this for you?
```

Original comment by `thomasp...@google.com` on 4 Dec 2015 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Closing issue out for now, please re-open if you need this prior to us rolling 
out the planned improvements to the import pipeline.
```

Original comment by `thomasp...@google.com` on 7 Dec 2015 at 7:46
- Changed state: **WontFix**
